packages/app/src/cli/services/logs.ts

```diff
@@ -131,3 +131,25 @@ async function prepareForLogs(commandOptions: LogsOptions): Promise<{
     localApp,
   }
 }
+
+function renderAppLogsConfigInfo(
+  appName: string,
+  storeFqdn: string,
+  storeFqdns?: string[],
+  configFile?: string,
+  org?: string,
+) {
+  const devStores = []
+  if (storeFqdns && storeFqdns.length > 0) {
+    storeFqdns.forEach((storeUrl) => devStores.push(storeUrl))
+  } else {
+    devStores.push(storeFqdn)
+  }
+  const body = formInfoBoxBody(appName, org, devStores, resetHelpMessage)
+  const fileName = configFile && getAppConfigurationFileName(configFile)
+
+  renderInfo({
+    headline: `${configFile ? `Using ${fileName} for default values:` : 'Using these settings:'}`,
```

Comment:
Is this message just for `app logs` or `app dev` as well? Based on our discussion with the CLI team, I think this headline should be updated for both commands.

Reply:
Oh, my bad. I thought I'm not changing anything about the `app dev` info box. To keep my changes limited to `app logs`. I will add this to `app dev` too.👍🏻